crypt.go

```diff
@@ -383,20 +384,20 @@ func activateWithTPMKey(tpm *TPMConnection, volumeName, sourceDevicePath, keyPat
 		}
 
 		switch {
-		case pinTries == 0 && k.AuthMode2F() == AuthModePIN:
+		case passphraseTries == 0 && k.AuthMode2F() == AuthModePIN:
 			return nil, nil, requiresPinErr
-		case pinTries == 0:
-			pinTries = 1
+		case passphraseTries == 0:
+			passphraseTries = 1
 		}
 
 		var sealedKey []byte
 		var authPrivateKey []byte
 
-		for ; pinTries > 0; pinTries-- {
+		for ; passphraseTries > 0; passphraseTries-- {
 			var pin string
 			if k.AuthMode2F() == AuthModePIN {
-				r := pinReader
-				pinReader = nil
+				r := passphraseReader
+				passphraseReader = nil
```

Comment:
Unrelated to renames, is this a precaution that the reader wouldn't be used again by accident?

Reply:
I'm not quite sure what was the original intention here. It's meant to fall back to use systemd-ask-password after consuming one passphrase.  But it's very specific behavior that's another reason to have some more control later via the Prompter if needed.